proposals/nnnn-stream-ontouchevent.md

```diff
@@ -0,0 +1,92 @@
+# Start Video Stream With OnTouchEvent Permission
+
+* Proposal: [SDL-NNNN](nnnn-stream-ontouchevent.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Awaiting review**
+* Impacted Platforms: [ iOS ]
+
+## Introduction
+
+This proposal is changing the behavior how the streaming video manager should include `OnTouchEvent` permission into the starting conditions.
+
+## Motivation
+
+SDL navigation applications stream the UI as a video to the vehicle head unit. In this case it's quite obvious that touch events are key for such applications to operate. However, for nav apps that connect to a head unit for the first time, the app ID is unknown and therefore it's not clear if the app is allowed to video stream or to receive touch events. Starting the video stream only upon `HMI_FULL` can lead to Core to reply with `NAK` on the start service request.
+
+## Proposed solution
+
+The library should wait to send the start service request until all conditions are met. The solution is to check the existence of `OnTouchEvent` in the latest `OnPermissionsChange` notification.
+
+Below code shows an example implementation in the class `SDLStreamingVideoLifecycleManager`. 
+
+First of all the manager should be able to store the last known permission status of OnTouchEvent in case that the permissions were provided before other conditions were met (e.g. permission update during HMI_NONE).
```

Comment:
```suggestion
First of all the manager should be able to store the last known permission status of OnTouchEvent in case the permissions were provided before other conditions were met (e.g. permission update during HMI_NONE).
```